app/src/main/java/com/sdp13epfl2021/projmag/model/ImmutableProject.kt

```diff
@@ -21,7 +22,8 @@ data class Failure<T>(val reason: String) : Result<T>()
     val bachelorProject: Boolean,
     val tags: List<String>,
     val isTaken: Boolean,
-    val description: String
+    val description: String,
+    val videoURI: Uri? = null
```

Comment:
Wouldn't it be better to be more generic and allow a list of videoUri ?

Reply:
Changed in [this commit](https://github.com/asfoury/projmag/pull/44/commits/a82e2f7c6c9baabae17d48da6984e62dc66810a3)